iothub_client/tests/common_longhaul/iothub_client_common_longhaul.c

```diff
@@ -399,14 +401,50 @@ static char* create_twin_desired_properties_update(const char* test_id, unsigned
     return result;
 }
 
+static void validate_internet_connectivity()
```

Comment:
Ot start of this function I'd recommend something like

```c
LogInfo("Error detected in test.  Verifying Internet connectivity available...");
```

This way when going through logs the time stamping is clearer if we have to correlate stuff and if the requset takes a long time so we know that we've lost say 60 seconds in a failed lookup of bing.com invoked by this test (which is fine) and not some random thread/busy loop sucking CPU away otherwise.